boards/posix/simple_process/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+zephyr_library()
```

Comment:
I dislike this "simple_process" name. In time it might not be that simple anymore, and in any case I think something like `boards/posix/process` would be better.

Reply:
I'm ok with calling it any sane name. But before changing it, I would prefer if you could first reach some agreement with other people who has a strong opinion about it so we don't need to change it back and forth.

Reply:
@nashif @galak can you weigh in?

Reply:
maybe just call it **native**